File: cpp/problem6.cpp
#include <iostream>
using namespace std;
//solution: 25164150

int eulerProblem6() {
    return (100*(100+1)/2)*(100*(100+1)/2)-100*(100+1)*(2*100+1)/6;
}

int main() {
    cout << eulerProblem6() << endl;
    return 0;
}

File: python/problem6.py
#!/usr/bin/python
#solution: 24174150

import math

def euler_problem_6():
    squares = 0
    sum = 0
    for i in range(100):
        sum += i
        squares += math.pow(i, 2)
    return int(math.pow(sum, 2) - squares)

print euler_problem_6()
